apps/extension/src/background/sdk/service.ts

```diff
@@ -18,14 +17,10 @@ export class SdkService {
     private readonly cryptoMemory: WebAssembly.Memory
   ) {}
 
-  static async init(localStorage: LocalStorage): Promise<SdkService> {
-    // Get stored chain
-    const chain = await localStorage.getChain();
-    // If chain is not stored, use default chain information
-    const rpc = chain?.rpc || chains.namada.rpc;
-
+  static async init(): Promise<SdkService> {
+    // Use fake RPC, the extension does not actually issue RPC requests:
+    const rpc = chains.namada.rpc;
```

Comment:
Should we add an empty string to prevent using it by accident in the future?

Reply:
Thanks! I cleaned this up some more, this service shouldn't need this value on init, then I just passed another `"NOT USED..."` string to initialize the SDK